src/solver/conv_bin_winoRxS_fused.cpp

```diff
@@ -275,7 +269,21 @@ ConvSolution ConvBinWinogradRxSFused::GetSolution(const FusionContext& context,
     };
     return result;
 }
+float ConvBinWinogradRxSFused::GetWti(const FusionContext&, const FusionDescription& problem) const
+{
+    // Negative values mean rough estimation of time
+    // Solvers with positive values are considered first priority
+    // If none found solvers with negative are used, -2.0f meaning a completely unknown value
+    // So lower absolute values mean higher priority
 
+    const auto conv_problem = problem.GetConvProblem(0, miopen::conv::Direction::Forward);
+    const auto x            = conv_problem.GetWeightsWidth();
+    const auto y            = conv_problem.GetWeightsHeight();
+    if(x == 3 && y == 3)
+        return -.01f;
+    else
+        return -.95f;
```

Comment:
```suggestion
        return wti_unknown / 200.f
    else
        return wti_unknown / 1.9f
```

We may wish to end up with a single place where unknown WTIs for all solvers are defined altogether (possibly as macros or global constexprs).